applicationinsights.ts

```diff
@@ -105,6 +125,21 @@ export function setup(setupString?: string) {
     return Configuration;
 }
 
+function _configureFromConfigFile() {
```

Comment:
You want to have this functionality for all telemetry clients not only default one